week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -1,41 +1,204 @@
 /*
-Create an object that acts like a water bottle. 
-It will need a volume key to store how full or empty the bottle is. 
-It will be 100 when full and 0 when empty. 
-Give your water bottle methods for filling it up, 
-drinking some of it, and emptying it.
+Create an object that acts a water bottle.
 
-We made a start on this for you here: 
+It will need a volume property to store how full or empty the bottle is. 
+Volume will be 100 when bottle is full and 0 when empty. 
+
+Give your water bottle methods for 
+  - filling it up
+  - pouring 10 unit of water into it
+    Note: You cannot over exceed the bottle capacity.
+  - drinking 10 unit from it
+    Note: You cannot drink more than its actual content.
+  - and telling if the bottle is full
+  - and telling if the bottle is empty
+
+We made a start on this for you here by giving the sceleton of our object.
+You have to implement the missing features according to the specification.
 */
 
+// Here is your starting point:
 let bottle = {
   volume: 0,
-  fill: function() {
-    // calling this function should make you bottles volume = 100;
+  fillUp: function () {
+    return (this.volume = 100);

```

Comment:
You don't need a return because you are not returning something. Instead, you are modifying a Key value.